s3/pkg/datastore/common/utils.go

```diff
@@ -12,16 +12,3 @@ func GetMd5FromCtx(ctx context.Context) (md5val string) {
 
 	return
 }
-
-func TrimQuot(in string) string {
```

Comment:
Why removing this? Not used or will not be used?

Reply:
Will not be used, cause etag got from AWS s3, Azure Blob, etc, start and end with double quotation marks, we will keep the same as them.

Reply:
Sorry, it should not be removed, I correct it now, thanks.